src/js/entities-service/entities/patient-fields.js

```diff
@@ -15,22 +14,12 @@ const _Model = BaseModel.extend({
   getPatient() {
     return this.getRelationship('_patient');
   },
-  isNew() {
-    // NOTE: This will treat the PATCH like a PUT
-    // We won't always have an ID, but never need to POST
-    return false;
-  },
   saveAll(attrs) {
     attrs = extend({}, this.attributes, attrs);
-    const patient = this.toRelation(attrs._patient);
-
-    // NOTE: sets the id instead of attrs.id due to how backbone's save works
-    /* istanbul ignore next: Currently not saving new fields, but would be important if we do */
-    if (!attrs.id) {
-      this.set({ id: uuid(`resource:field:${ attrs.name.toLowerCase() }`, patient.id) });
```

Comment:
this issue here was that `toRelation` returns `{ data: { id, type }`
I was a little surprised this wasn't caught by a test, but I missed the above ignore.  It turns out because of our model caching, setting the id really messed with tests..  but because we _can_ always determine an id with v5 uuid we don't have use for a bit of the workaround logic in here anyhow